src/eval.py

```diff
@@ -122,18 +135,72 @@ def evaluate(
                     .cpu()
                     .numpy()
                 )
-                preds = finetuned_model.predict(encodings)
+                preds = finetuned_model.predict(np.nan_to_num(encodings, nan=0.0))
```

Comment:
Do we want this to happen? If we are returning NaNs it might be worth digging into (perhaps in a separate issue / PR).

Reply:
Yup good catch, I was getting nans but it might be one of my GPUs (was also happening on presto-v2). I'm testing it on another GPU and I'll take this out